src/ontology/cl-edit.owl

```diff
@@ -7798,7 +7804,6 @@ AnnotationAssertion(oboInOwl:inSubset obo:CL_0000548 <http://purl.obolibrary.org
 AnnotationAssertion(rdfs:label obo:CL_0000548 "animal cell"^^xsd:string)
 EquivalentClasses(obo:CL_0000548 ObjectIntersectionOf(obo:CL_0000003 ObjectSomeValuesFrom(obo:RO_0002160 obo:NCBITaxon_33208)))
 SubClassOf(obo:CL_0000548 obo:CL_0000255)
-SubClassOf(obo:CL_0000548 ObjectSomeValuesFrom(obo:RO_0002160 obo:NCBITaxon_33208))
```

Comment:
Was this intended?

Reply:
Thank you for noticing @bvarner-ebi , I have fixed it